hypertrace-trace-enricher/trace-reader/src/main/java/org/hypertrace/trace/reader/entities/AvroEntityConverter.java

```diff
@@ -0,0 +1,134 @@
+package org.hypertrace.trace.reader.entities;
+
+import static java.util.Objects.nonNull;
+
+import io.reactivex.rxjava3.core.Observable;
+import io.reactivex.rxjava3.core.Single;
+import java.util.Map;
+import java.util.Map.Entry;
+import javax.annotation.Nonnull;
+import org.hypertrace.core.datamodel.AttributeValue;
+import org.hypertrace.core.datamodel.Attributes;
+import org.hypertrace.core.datamodel.Entity;
+import org.hypertrace.entity.data.service.v1.AttributeValueList;
+import org.hypertrace.entity.data.service.v1.AttributeValueMap;
+import org.hypertrace.entity.data.service.v1.Value;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+class AvroEntityConverter {
```

Comment:
While I might be wrong, I feel all methods here can be static. How about that?

Reply:
Sure - I have a pretty prejudiced dislike of statics, but it's more rooted in other languages.